components/collective-page/hero/HeroAvatar.js

```diff
@@ -137,12 +142,32 @@ const HeroAvatar = ({ collective, isAdmin, intl, handleHeroMessage }) => {
               <EditableAvatarContainer isDragActive={isDragActive}>
                 <EditOverlay borderRadius={borderRadius}>
                   {!isDragActive && (
-                    <StyledButton buttonSize="tiny" minWidth={120}>
-                      <Camera size={12} />
-                      <Span ml={2} css={{ verticalAlign: 'center' }}>
-                        <FormattedMessage id="HeroAvatar.Edit" defaultMessage="Edit logo" />
-                      </Span>
-                    </StyledButton>
+                    <Box>
+                      <StyledButton buttonSize="tiny" minWidth={120}>
+                        <Camera size={12} />
+                        <Span ml={2} css={{ verticalAlign: 'center' }}>
+                          <FormattedMessage id="HeroAvatar.Edit" defaultMessage="Edit logo" />
+                        </Span>
+                      </StyledButton>
+                      {!(
+                        collective.imageUrl.includes(`/${collective.slug}/logo`) ||
+                        collective.imageUrl.includes(`/${collective.slug}/avatar`)
+                      ) && (
```

Comment:
There's a simpler way to check that

```suggestion
                      {!collective.image && (
```